src/renderer/src/components/steps/GeneratingApplication.jsx

```diff
@@ -61,9 +62,20 @@ const GeneratingApplication = React.forwardRef(({ onBack, onRestartProcess, useV
   }
 
   function onClickCopyLogs () {
+    setLogsCopied(true)
     let str = ''
     npmLogs.forEach(log => (str += `${log.message}\r\n`))
     navigator.clipboard.writeText(str)
+    setTimeout(() => {
```

Comment:
Why this? `writeText` is async. https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText

Reply:
I added this setTimeout to change the icon